clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy

```diff
@@ -48,16 +48,22 @@ import java.security.Security
 @EnableScheduling
 class Main extends SpringBootServletInitializer {
 
-  static final Map<String, String> DEFAULT_PROPS = [
+  private static final Map<String, String> DEFAULT_PROPS = [
     'netflix.environment'    : 'test',
     'netflix.account'        : '${netflix.environment}',
     'netflix.stack'          : 'test',
+    'spring.profiles.active' : '${netflix.environment},local',
     'spring.config.location' : '${user.home}/.spinnaker/',
-    'spring.application.name': 'clouddriver',
-    'spring.config.name'     : 'spinnaker,${spring.application.name}',
-    'spring.profiles.active' : '${netflix.environment},local'
+    'spring.config.name'     : 'spinnaker,${spring.application.name}'
   ]
 
+  static final Map<String, String> BOOTSTRAP_SYSTEM_PROPS = [
+    'spring.application.name'               : 'clouddriver',
+    'spring.cloud.bootstrap.location'       : '${user.home}/.spinnaker/',
+    'spring.cloud.bootstrap.name'           : 'spinnaker-config,${spring.application.name}-config',
```

Comment:
I'd suggest avoid a dash in these names since there are semantics around `-profile` and this could be interpreted as configuration for spinnaker or application in the `config` profile (something like  `spinnakerconfig,${spring.application.name}config` avoids that collision case)

Reply:
Agreed on the confusion with `-profile`. I've struggled to find something I like, but `spinnakerconfig,${spring.application.name}config` works for me.